networking/eth2/src/main/java/tech/pegasys/artemis/networking/eth2/peers/Eth2Peer.java

```diff
@@ -56,7 +56,7 @@ public void updateStatus(final PeerStatus status) {
   }
 
   public void subscribeInitialStatus(final InitialStatusSubscriber subscriber) {
-    initialStatus.thenAccept(subscriber::onInitialStatus);
+    initialStatus.finish(subscriber::onInitialStatus);
```

Comment:
Should we add something like `finishAndIgnoreErrors`?  This isn't really a case we need to log at error level.  We should have another process that's handling status requests.

Reply:
I see there's also an `ignoreFuture` utility method, and the `@SuppressWarnings("FutureReturnValueIgnored")` annotation.  These kind of do the same thing right?  When would we use each style?

Reply:
Yes `ignoreFuture` and suppress warnings are essentially the same thing.  Mostly I like the more explicit `ignoreFuture` but when dealing directly with Netty you get a *lot* of futures back and Netty already has error handling so the exceptions won't get lost.  It's misleading in that case to use `ignoreFuture` so in the couple of places where we deal with a lot of Netty and no other futures I wound up using the suppress warnings version.

In this particular case, there can't ever be an exception - the future is either completed successfully or not completed.  If we really want to finish and ignore errors, or log at something other than error level, we should probably be explicit about it and use:
```
initialStatus.finish(
  subscriber::onInitialStatus,
  error -> LOG.debug("Oh well", error));
```

I'm very much treating it like the uncaught exception handler - ideally it should never be invoked because you should have handled exceptions properly at some point in your code but it's a backstop so that if something unexpected happens the error is printed.